rest-fights/src/main/resources/application.properties

```diff
@@ -14,15 +14,26 @@ quarkus.http.test-port=0
 ## Mongo
 quarkus.mongodb.metrics.enabled=true
 quarkus.mongodb.database=fights
-
+#quarkus.mongodb.connection-string=mongodb://localhost:52003
+#quarkus.mongodb.credentials.username=superfight
+#quarkus.mongodb.credentials.password=superfight
 ## Liquibase Mongo
 quarkus.liquibase-mongodb.migrate-at-start=true
 
 ## CORS
 quarkus.http.cors=true
-quarkus.http.cors.origins=*
-
+quarkus.http.cors.origins=http://localhost:8082,http://localhost:8080,http://localhost:8086
+quarkus.http.cors.access-control-allow-credentials=true
+org.eclipse.microprofile.rest.client.propagateHeaders=Cookie
+#org.eclipse.microprofile.rest.client.propagateHeaders=Authorization
+quarkus.log.category."org.jboss.resteasy.reactive.logging".level=DEBUG
```

Comment:
This shouldn't have been committed

Reply:
`org.eclipse.microprofile.rest.client.propagateHeaders=Cookie` so if I have `@RegisterClientHeaders` I don't need to specify the parameter for the cookie? 

Also commented above about this one: `quarkus.http.cors.access-control-allow-credentials=true`